RELEASE/scripts/autoscend.ash

```diff
@@ -1,4 +1,4 @@
-since r20655;	// min mafia revision needed to run this script. Last update: questM05Toot tracking will correct to finished when refreshing quests
+since r20730;	// min mafia revision needed to run this script. Last update: questM05Toot tracking will correct to finished when refreshing quests
```

Comment:
not strictly necessary but we like to keep the comment here up to date with the change from that mafia revision too as there's usually significant details in why it was updated to that version.